components/editor/modules/article/collection.js

```diff
@@ -0,0 +1,183 @@
+import MarkdownSerializer from 'slate-mdast-serializer'
+import { Label, Field, Checkbox } from '@project-r/styleguide'
+import { Block } from 'slate'
+
+import { buttonStyles, matchBlock } from '../../utils'
+import { createRemoveEmptyKeyHandler } from '../../utils/keyHandlers'
+
+import injectBlock from '../../utils/injectBlock'
+import UIForm from '../../UIForm'
+
+const getNewBlock = options => {
+  const { headerModule, articleGroupModule } = getSubmodules(options)
+  return () => Block.create({
+    kind: 'block',
+    type: options.TYPE,
+    nodes: [
+      Block.create({
+        kind: 'block',
+        type: headerModule.TYPE
+      }),
+      articleGroupModule.helpers.newItem()
+    ]
+  })
+}
+
+export const getData = data => ({
+  membersOnly: true,
+  unauthorizedText: '',
+  ...(data || {})
```

Comment:
nit: `|| {}` not needed, spreading undefined is supported (behaviour of object spread is the same as `Object.assign`)